README.md

```diff
@@ -180,6 +181,9 @@ curated components, see the [Registry](https://aframe.io/aframe-registry).
 - [Sprite Component](https://github.com/tizzle/aframe-sprite-component) - Creates bitmap images that always face the camera
 - [UI Components](https://github.com/gmarty/aframe-ui-components) - A collection of components related to UI
 - [Pipe](https://github.com/jellix79/aframe-pipe-component) - A pipe with inner and outer radius
+- [Persist Component](https://github.com/Utopiah/aframe-persist-component) - Use localStorage to make data persist over experiences
+- [Gradient Sky](https://github.com/zcanter/aframe-gradient-sky) - Use a gradient on sky
+- [Mouse Cursor Component](https://github.com/mayognaise/aframe-mouse-cursor-component) - Add interaction to the mouse cursor
```

Comment:
Could be a bit more descriptive (uses hardware mouse, raycasts from XY position on canvas)